### ISSUE TITLE
Uncaught Error: Cannot find module

### ISSUE THREAD

**Issue by sasxa** [wallabyjs/public#1390]
### Issue description or question
I'm getting an error `Cannot find module`:
```
​​​​​​​​​​​​Uncaught Error: Cannot find module '.\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)​​
​​at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)​​
​​at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​Uncaught Error: Cannot find module '166' (c:\dev\projects\webapps\node_modules\@nrwl\nx\testing.js)​​
​​at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
```

Some other files appear in log sometimes. I checked, files are there. Any idea why this is happening?

Here's the console output:
```
...
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:48 GMT wallaby:middleware Serving /libs/utils/src/rxjs.spec.js.wbp.js from cache

​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:48 GMT wallaby:workers Sandbox (active) [7il76] error: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '.\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:48 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js), stack: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [7il76] error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js), stack: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [7il76] Loaded 1 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [7il76] Test executed: should work

​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [42lez] error: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '.\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js), stack: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [42lez] error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js), stack: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [42lez] Loaded 0 test(s)

​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [nf6dz] error: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '.\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js), stack: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [7il76] Run 1 test(s), skipped 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [7il76] Sandbox is responsive, closing it
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [nf6dz] error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js), stack: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1

​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [0q2cd] error: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '.\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js), stack: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [nf6dz] Loaded 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [0q2cd] error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js), stack: Uncaught Error: Cannot find module '154' (c:\dev\projects\webapps\node_modules\@angular\core\esm5\testing.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1

​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [fotet] error: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '.\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js), stack: Uncaught Error: Cannot find module 'C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js' (C:\Users\sasxa\.vscode-insiders\extensions\WallabyJs.wallaby-vscode-1.0.65\projects\ca4924c20c8f3b08\instrumented\wallaby.test.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Sandbox (active) [fotet] error: Uncaught Error: Cannot find module '166' (c:\dev\projects\webapps\node_modules\@nrwl\nx\testing.js)
​​[Error] Runtime error: Uncaught Error: Cannot find module '166' (c:\dev\projects\webapps\node_modules\@nrwl\nx\testing.js)​​
​​[Error] at http://localhost:51517/wallaby-webpack.js?1510656526380:1​​
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module '166' (c:\dev\projects\webapps\node_modules\@nrwl\nx\testing.js), stack: Uncaught Error: Cannot find module '166' (c:\dev\projects\webapps\node_modules\@nrwl\nx\testing.js)
​​​​​[Info]​​​​​ at http://localhost:51517/wallaby-webpack.js?1510656526380:1

​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [0q2cd] Loaded 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [fotet] Loaded 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [42lez] Run 0 test(s), skipped 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [42lez] Sandbox is responsive, closing it
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [nf6dz] Run 0 test(s), skipped 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [nf6dz] Sandbox is responsive, closing it
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [0q2cd] Run 0 test(s), skipped 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [fotet] Run 0 test(s), skipped 0 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [0q2cd] Sandbox is responsive, closing it
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [fotet] Sandbox is responsive, closing it
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:49 GMT wallaby:workers [tew6v] Loaded 79 test(s)
​​​​​[Info]​​​​​ Tue, 14 Nov 2017 10:48:50 GMT wallaby:workers [tew6v] Test executed: should create
...
````

### Wallaby.js configuration file

```javascript
module.exports = function (wallaby) {
  var webpackPostprocessor = wallabyWebpack({
    entryPatterns: [ 'wallaby.test.js', ...patterns ],

    module: {
      loaders: [...],

    emitModulePaths: true,
    resolve: {
      alias: {
        '@library': path.join(wallaby.projectCacheDir, 'libs')
      },
      extensions: [ '.js', '.ts' ],
      modules: [
        path.join(wallaby.projectCacheDir, 'apps'),
        path.join(wallaby.projectCacheDir, 'libs'),
        'node_modules'
      ]
    },
  });

  return {
    files: [
      { pattern: 'wallaby.test.ts', load: false },
      { pattern: '**/node_modules/**/*', ignore: true },
      {
        pattern: 'apps/**/*.+(ts|css|less|scss|sass|styl|html|md|json|svg)',
        load: false
      },
      { pattern: 'apps/**/*.d.ts', ignore: true },
      { pattern: 'apps/**/*spec.ts', ignore: true },
      {
        pattern: 'libs/**/*.+(ts|css|less|scss|sass|styl|html|md|json|svg)',
        load: false
      },
      { pattern: 'libs/**/*.d.ts', ignore: true },
      { pattern: 'libs/**/*spec.ts', ignore: true },
      { pattern: '**/dist/**', ignore: true },
    ],

    tests: [
      { pattern: 'apps/**/*spec.ts', load: false },
      { pattern: 'apps/**/e2e/**/*', ignore: true },
      { pattern: 'libs/**/*spec.ts', load: false },
      { pattern: 'libs/**/e2e/**/*', ignore: true },
    ],
  };
};

```

### Code editor or IDE name and version

Visual Studio Code 
1.19.0-insider
73830a36872aa1c168c085dad8cc6e2e1a05ff45
x64

### OS name and version

Windows

**Comment by ArtemGovorov** [comment]
Are you getting the errors during the initial test run?

Either way, it's hard to tell without a sample repo. Could you please create and share one where we could reproduce the configuration issue - happy to have a look. Hopefully a stripped down version of `/libs/utils/src/rxjs.spec.ts` (with the rest of config files), will be enough to reproduce the issue.

**Comment by sasxa** [comment]
Yes, error happens during initial run.

I'll try to reproduce the issue, with setup similar to mine... Wallaby used to work normally, but I didn't use it for some time and did some work since then, but now I'm getting this error.

**Comment by ArtemGovorov** [comment]
@sasxa Cool, thanks! Looking forward to look into the repo.

Just in case, to rule out possible environment issues, could you please quickly [clone this sample repo](https://github.com/wallabyjs/ngCliWebpackSample/), run `npm i` and see if it's working for you? It has a structure similar to yours.

**Comment by sasxa** [comment]
@ArtemGovorov I couldn't reproduce the issue, but I managed to "avoid" it by setting `workers.initial = 1`. With that wallaby is working. No idea why (: I was trying to pinpoint where exactly error was happening, that's why I changed this setting...

**Comment by ArtemGovorov** [comment]
Normally wallaby runs your tests in few sandboxes in parallel. With the setting you make wallaby run all your tests in one sandbox. 

Given it works in one sandbox, it probably means that some of your tests depend on other tests/setup in some way. So when all tests are executed at once it works, but when some of your tests are executed in isolation, then it works in a sandbox where everything is set up correctly, but fails in sandboxes that have tests that are unable to run in isolation from other tests. Hope it makes sense.

So in order to reproduce it, you'll need to include more test files into the repro. 

Also you may try adding your spec files one by one to the `tests` list and run them in isolation (from other specs). This way you'll be able to find out which spec file is unable to run by itself (without other spec files)

**Comment by ArtemGovorov** [comment]
Closing the issue due to lack of activity. if it's still a valid issue for you, please try the steps listed above, or/and create a sample repo demonstrating the issue - we are happy to have a look into it.

**Comment by kevinbeal** [comment]
This was the issue for me. One test file was importing from another. Once I moved that import into another file to import into both, the issue was resolved.

To see which file I was importing and was the issue, I looked at the number following "Cannot find module" and looked at the bundled file (using webpack) which was located here: http://localhost:58931/wallaby-webpack.js?1537371974555:1

I looked for the number (i.e. 780) and saw it was the value whose key was a file path. That file was the one I was importing.
